lib/venue.rb

```diff
@@ -1,2 +1,28 @@
 class Venue
+  def initialize(name, number)
+    @name = name
+    @number = number
+    @patrons = []
+
+def name
+  @name
+end
+
+def number
+  @number
+end
+
+def add_patron(patron)
+  @patrons << patron
+end
+
+def patrons
+  @patrons
+end
+
+def yell_at_patrons(patron)
```

Comment:
Does the `yell_at_patrons` take an argument?
```
venue.yell_at_patrons
=> ["MIKE", "MEGAN", "BOB"]
```
What data type is `@patrons`? You will need to iterate over the `@patrons` array and then what would you need to do to make all of the names upcased?

I would review the #each lesson https://backend.turing.edu/module1/lessons/each